content/views.py

```diff
@@ -583,11 +584,25 @@ def list(self, request, pk=None, *args, **kwargs):
 
         return Response(serializer.data)
 
+# ====== #
+# Badges #
+# ====== #
+
+class BadgesView(GenericAPIView):
+    def get(self, request, *args, **kwargs):
+        queryset = Badge.objects.all()
+        urls = []
+        for badge in queryset:
+            urls.append(request.build_absolute_uri(badge.image.file.url))
```

Comment:
The image field on Badge can be None. Check if this fails if a Badge image is not set.